src/mango/src/mango_idx.erl

```diff
@@ -44,7 +44,7 @@
     to_json/1,
     delete/4,
     get_usable_indexes/3,
-    get_idx_selector/1
+    get_idx_partial_filter_selector/1
```

Comment:
rename to `get_partial_filter_selector` - that it is in the `mango_idx` module implies that it relates to an index